nixos/modules/services/web-apps/kasmweb.nix

```diff
@@ -0,0 +1,342 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
```suggestion
```

Well, that thing of abusing `with lib;` should be avoided. 

https://nix.dev/anti-patterns/language#with-attrset-expression

Yes, you will see many things like this on the whole Nixpkgs codebase, because the codebase is old and we are little by little cleaning it up.

P.S.: by removing this line, some code below will become invalid "as is". Just prefix `lib.` in the missing functions.

Reply:
I'll fix this.